src/components/OverflowableText/overflowable-text.jsx

```diff
@@ -15,6 +15,8 @@ const overflowStyle = {
         whiteSpace: 'pre',
         textOverflow: 'ellipsis',
         overflow: 'hidden',
+        lineHeight: 'initial',
+        verticalAlign: 'middle',
```

Comment:
make a wrapper instead of customizing the very generic component

Reply:
I found a way to stylize it on gridexplore side